### PR TITLE
Update SDK and Extension nugets 

### PR DESCRIPTION
**Pull request by MikeStall**
Update SDK to 10646
Update Extension to 10427

No functional changes.

### CODE REVIEW

File: src/WebJobs.Script.WebHost/Web.config [hunk 1]
```diff
@@ -148,7 +148,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.IO.Compression" publicKeyToken="b77a5c561934e089" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0" />
```
Comment (fabiocav) [inline src/WebJobs.Script.WebHost/Web.config]: Need to undo this change. We need to keep the redirect pushing it back to the built in version (4.0.0)

File: src/WebJobs.Script/app.config [hunk 2]
```diff
@@ -89,7 +89,7 @@
       <dependentAssembly>
         <assemblyIdentity name="System.IO.Compression" publicKeyToken="b77a5c561934e089" culture="neutral" />
         <!--Explicitly redirecting back to 4.0.0. See https://github.com/dotnet/corefx/issues/15693 for details-->
-        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0" />
```
Comment (fabiocav) [inline src/WebJobs.Script/app.config]: Need to undo this. Same issue mentioned above.

File: src/WebJobs.Script.WebHost/Web.config (hunk 1 above)
Comment (MikeStall) [inline src/WebJobs.Script.WebHost/Web.config]: Ok. Just curious, what's going on here? VS makes this update by default when I updated the sdk/extension packages. .

Comment (MikeStall) [inline src/WebJobs.Script.WebHost/Web.config]: Ah, I see: https://github.com/dotnet/corefx/issues/15693
